addons/sourcemod/scripting/lilac/lilac_convar.sp

```diff
@@ -17,24 +17,49 @@
 */
 
 
-/* Basic query list. */
-static char query_list[][] = {
-	"sv_cheats",
-	"r_drawothermodels",
-	"mat_wireframe",
-	"snd_show",
-	"snd_visualize",
-	"mat_proxy",
-	"r_drawmodelstatsoverlay",
-	"r_shadowwireframe",
-	"r_showenvcubemap",
-	"r_drawrenderboxes",
-	"r_modelwireframedecal"
-};
-
 static int query_index[MAXPLAYERS + 1];
 static int query_failed[MAXPLAYERS + 1];
 
+/* Structure to store convar validation rules */
+enum struct ConvarRule {
+    char name[32];
+    int expected_value;
+    bool is_minimum;  // If true, value must be >= expected_value
+}
+
+/* Basic query list. */
+static ConvarRule convar_rules[] = {
```

Comment:
Maybe put this in a config file directly

Reply:
I'm not sure, we want to avoid users to put bad cvars. That' why it's hard coded.

Reply:
we dont trust server managers?